Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -329,10 +330,18 @@ public void onBackPressed()
 		}
 		else
 		{
-			mEmulationFragment.stopEmulation();
-			exitWithAnimation();
+			if(backPressedOnce)
```

Comment:
Not sure about the Android code style, but the rest of the file seems to have a space between the `if` and the opening parenthesis.